suite/tests/CMakeLists.txt

```diff
@@ -2170,7 +2170,7 @@ if (CLIENT_INTERFACE)
     "" "" "")
 
   tobuild_ci(client.option_parse client-interface/option_parse.cpp
-    "-x;4;-y;quoted string;-z;first;-z;single quotes -dash --dashes;-front;value;-y;accum;-front2;value2;-no_flag;-takes2;1_of_4;2_of_4;-takes2;3_of_4;4_of_4;-val_sep;v1.1 v1.2;-val_sep;v2.1 v2.2;-val_sep2;v1;v2;-val_sep2;v3;v4;-large_bytesize;9999999999"
+    "-l;-4;-ll;-749882561;-ul;4;-ull;823636799;-x;4;-y;quoted string;-z;first;-z;single quotes -dash --dashes;-front;value;-y;accum;-front2;value2;-no_flag;-takes2;1_of_4;2_of_4;-takes2;3_of_4;4_of_4;-val_sep;v1.1 v1.2;-val_sep;v2.1 v2.2;-val_sep2;v1;v2;-val_sep2;v3;v4;-large_bytesize;9999999999"
```

Comment:
It seems best to pass values that only a long long could hold.  These are both under 1 billion and so a regular long (at least for ILP64) will hold it.  I would add at least one more digit.